release-notes/vJanuary.md

```diff
@@ -143,7 +143,7 @@ We also added two new actions, "Find Next Selection" (`kb(editor.action.nextSele
 
 ### Input Handling
 
-We changed input handling in the editor to allow software such as [UniKey](http://www.unikey.org) (used for Vietnamese input) or [AutoHotKey](https://www.autohotkey.com) (used for keyboard automation) to work with VS Code.
+We changed input handling in the editor to allow software such as [UniKey](https://www.unikey.org) (used for Vietnamese input) or [AutoHotKey](https://www.autohotkey.com) (used for keyboard automation) to work with VS Code.

```

Comment:
We don't maintain the blogs and release notes (they were accurate at a point in time) but we shouldn't make changes that would break them.